conans/test/functional/command/search_test.py

```diff
@@ -1076,3 +1078,193 @@ class Test(ConanFile):
             client.run("search Test/0.1@lasote/testing  %s --outdated" % remote)
             self.assertIn("os: Windows", client.user_io.out)
             self.assertNotIn("os: Linux", client.user_io.out)
+
+
+@unittest.skipUnless(TestClient().revisions,
```

Comment:
Would like to see a test with the command ``conan search --revisions`` failing in the client when parameter ``--revisions`` is not available due to ``CONAN_CLIENT_REVISIONS_ENABLED=``